### ISSUE TITLE
Shortcuts suggestion

### ISSUE THREAD

**Issue by AmrSaber** [excalidraw/excalidraw#3954]
I have a suggestion for the shortcut for the line (P), normally when you hold the mouse and have a hand on the keyboard, you hold the mouse with your right hand, and you have your left hand on the keyboard (for right-handed people at least).

All the tools shortcuts are in the region for the left-hand on the keyboard, except the line shortcut (P), it's on the right-hand side of the keyboard, which is inconvenient and forces you to look for the letter and press it with left hand or leave the mouse and use the right hand's muscle memory to press the button with the right hand.

So if the shortcut was moved to the left-hand side of the keyboard, it will make its usage a lot more convenient.

For reference, here is an image of the keyboard's
<img width="856" alt="keyboard_fingers" src="https://user-images.githubusercontent.com/16343060/131658318-5d898f64-99cd-407f-8fd9-07e597b5f6e1.png">
 keys distribution for the right and left hands...

**Comment by hzh-git** [comment]
I agree with AmrSaber, and I hope all shortcut keys can be customed eventually, just like other advanced software.
